Update addTopicButtons.js

TechnishGeredt opened the pull request:
Updated addTopicButtons.js with Quote In Comments button

TechnishGeredt committed:
Update addTopicButtons.js

Updated addTopicButtons.js with Quote In Comments button

chucem commented on extension/js/addTopicButtons.js:
```diff
@@ -376,3 +376,108 @@ addBtn();
 
 
+async function fetchTopicMapping() {
+    const url = `https://sheets.googleapis.com/v4/spreadsheets/1bziJ_h6bIRBXFSBJHT3aBsb13-QrEnrSjjy_-IS3FFw/values/CommentMapping!A2:B?key=***API_KEY***`;
```
this URL doesn't seem valid

TechnishGeredt committed:
Update addTopicButtons.js with no API key required

An API key is no longer required, instead there is a new direct CVS file link to a Google Sheets shared page. And also the mapping file is no longer getting downloaded with each page refresh.

chucem committed:
Merge branch 'master' into patch-1
